src/Sylius/Component/Variation/Model/OptionValueInterface.php

```diff
@@ -11,13 +11,13 @@
 
 namespace Sylius\Component\Variation\Model;
 
+use Sylius\Component\Product\Model\OptionValueTranslationInterface;
 use Sylius\Component\Resource\Model\CodeAwareInterface;
-use Sylius\Component\Resource\Model\ResourceInterface;
 
 /**
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
-interface OptionValueInterface extends ResourceInterface, CodeAwareInterface
+interface OptionValueInterface extends CodeAwareInterface, OptionValueTranslationInterface
```

Comment:
This change should be reverted. You shouldn't couple `OptionValueInterface` with `OptionValueTranslationInterface`. You can check how options are implemented to get a clue.